fastly/block_fastly_service_v1_dictionary_test.go

```diff
@@ -58,42 +68,47 @@ func TestAccFastlyServiceV1_dictionary(t *testing.T) {
 				Config: testAccServiceV1Config_dictionary(name, dictName, backendName, domainName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckServiceV1Exists("fastly_service_v1.foo", &service),
-					testAccCheckFastlyServiceV1Attributes_dictionary(&service, name, dictName, false),
+					testAccCheckFastlyServiceV1Attributes_dictionary(&service, &dictionary, name, dictName, false),
 				),
 			},
-		},
-	})
-}
-
-func TestAccFastlyServiceV1_dictionary_write_only(t *testing.T) {
-	var service gofastly.ServiceDetail
-	name := fmt.Sprintf("tf-test-%s", acctest.RandString(10))
-	dictName := fmt.Sprintf("dict %s", acctest.RandString(10))
-	backendName := fmt.Sprintf("%s.aws.amazon.com", acctest.RandString(3))
-	domainName := fmt.Sprintf("fastly-test.tf-%s.com", acctest.RandString(10))
-
-	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckServiceV1Destroy,
-		Steps: []resource.TestStep{
 			{
-				Config: testAccServiceV1Config_dictionary_write_only(name, dictName, backendName, domainName),
+				Config: testAccServiceV1Config_dictionary(name, updatedDictName, backendName, domainName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckServiceV1Exists("fastly_service_v1.foo", &service),
-					testAccCheckFastlyServiceV1Attributes_dictionary(&service, name, dictName, true),
+					testAccCheckFastlyServiceV1Attributes_dictionary(&service, &dictionary, name, updatedDictName, false),
+				),
+			},
+			{
+				Config: testAccServiceV1Config_dictionary(name, updatedDictName, backendName, domainName),
+				Check:  testAccAddDictionaryItems(&dictionary),
+			},
+			{
+				Config:      testAccServiceV1Config_dictionary(name, dictName, backendName, domainName),
+				ExpectError: regexp.MustCompile(""),
```

Comment:
Should we not get an error like we do with the ACL tests?

e.g.

```go
regexp.MustCompile("Cannot delete.*list is not empty.*")
```

Reply:
Oh that's a facepalm right there! Not sure where that's gone 😂

Reply:
Rerunning the test with this added back in 🤦🏻‍♂️